.gitlab/build-rocm.sh

```
@@ -0,0 +1,446 @@
+        cmake --build . --parallel "$NPROC" --target azure-storage-blobs azure-identity && \
+        $SUDO cmake --install sdk/core && \
+        $SUDO cmake --install sdk/storage/azure-storage-common && \
+        $SUDO cmake --install sdk/storage/azure-storage-blobs && \
+        $SUDO cmake --install sdk/identity
+    )
+fi # PRE_INSTALLED_ENV
+
+if [ -n "$PRE_INSTALLED_UCX_ENV" ]; then
+    echo "PRE_INSTALLED_UCX_ENV is set, skipping UCX compilation"
+else
+    UCCL_COMMIT_SHA="0cdb740cf369a4f4dd63b9b773c8937f187b179a"
```

Shellcheck (0.11.0)

[warning] UCCL_COMMIT_SHA appears unused. Verify use (or export if used externally).

(SC2034)